src/main/java/journeymap/client/render/map/GridRenderer.java

```diff
@@ -74,14 +73,13 @@ public class GridRenderer
     private FloatBuffer winPosBuf;
     private FloatBuffer objPosBuf;
 
-    public GridRenderer(int gridSize)
```

Comment:
Leave the API as is, use wrappers to clamp the inputs to a new scheme.

Reply:
I did mean to keep public API breakage to an absolute minimum. You never know, where it would be called, and you can't possibly fix every dependent code you don't control.

Reply:
If it works fine in gtnh pack it was be removed